### DISCUSSION TITLE
API that returns info about CIS IGs and/or banchmarks (CSPM)

### DISCUSSION THREAD

**Discussion by threathunter89** [CrowdStrike/falconpy#1248] [Q&A]
Hi folks, I am working on the CSPM module. I want to pull data that will help me v API call that would return this information. 

The closest I got is get_policy_settings, which returns id, benchmark short, and recommendation number. Is there any other method that can provide what I need?

**Comment by crowdstrikedcs** [comment]
Hi @threathunter89 thanks for the question!. Could you let us know a bit more detail on the data points you are looking to collect? If there is a page in the Falcon UI where this data exists today that would be helpful to provide.

**Reply by threathunter89** [reply]
Cloud Security>IOM>Compliance 
<img width="332" alt="{4928003E-395F-42D4-B13D-B2A7FDA2DEDA}" src="https://github.com/user-attachments/assets/a2358ec7-5cb5-4b5f-a29e-aad4d45060cb">

And when you click on CIS, a separate page opens and it has mappings to IGs (for CIS version 8 and higher) 
<img width="776" alt="image" src="https://github.com/user-attachments/assets/89175e29-187c-40f5-8502-0c7df691fdf9">

**Reply by threathunter89** [reply]
So I am looking for an API that will return details IOMs that include CIS control version and finding (in example v8 3.11) and which IG belongs to (on screenshots Applicable Profiles)
